Enabled collapsible panel (#225)
mckinsey/vizro: vizro-core/changelog.d/20240112_163629_nadija_ratkusic_graca_enable_collapsible_panel.md

<!--
A new scriv changelog fragment.

Uncomment the section that is right (remove the HTML comment wrapper).
-->

<!--
### Highlights ✨

- A bullet item for the Highlights ✨ category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))

-->
<!--
### Removed

- A bullet item for the Removed category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))

-->

### Added

> - Implemented the functionality allowing users to collapse or expand the side panel with a button click. ([#225](https://github.com/mckinsey/vizro/pull/225))

Reviewer: ```suggestion
-  Enable the side panel to collapse/expand via a button that is present by default ([#225](https://github.com/mckinsey/vizro/pull/225))
```
